packages/firestore/src/local/memory_remote_document_cache.ts

```diff
@@ -177,7 +178,9 @@ export class MemoryRemoteDocumentCache implements RemoteDocumentCache {
     return PersistencePromise.resolve(changedDocs);
   }
 
-  newChangeBuffer(): RemoteDocumentChangeBuffer {
+  newChangeBuffer(options: {
+    createSentinelDocumentsToTrackDeletes: boolean;
```

Comment:
Perhaps there should be a comment indicating we explicitly ignore this.

Reply:
Added. This makes me thing we should never have added the change log to the MemoryRemoteDocumentCache.